modules/osdk-cli-reference-alpha.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-cli-reference.adoc
+
+[id="osdk-cli-reference-alpha_{context}"]
+= alpha
+
+The `operator-sdk alpha` command is used to run an alpha subcommand.
+
+== scorecard
+
+The `alpha scorecard` subcommand runs the scorecard tool to validate an Operator bundle and provide suggestions for improvements. The command takes one argument, either a bundle image or directory containing manifests and metadata. If the argument holds an image tag, the image must be present remotely.
+
+.`scorecard` flags
+[options="header",cols="1,3"]
+|===
+|Flag |Description
+
+|`-c`, `--config` (string)
+|Path to scorecard configuration file.
+
+|`-h`, `--help`
+|Help output for the `scorecard` command.
+
+|`--kubeconfig` (string)
+|Path to `kubeconfig` file.
+
+|`-L`, `--list`
+|List which tests are available to run.
+
+|`-n`, `--namespace` (string)
+|Namespace in which to run the test images. Default: `default`.
```

Comment:
In the 4.9 documents for [scorecard](https://docs.openshift.com/container-platform/4.9/operators/operator_sdk/osdk-cli-ref.html#osdk-cli-ref-scorecard_osdk-cli-ref), the default value is presented as "The default value is `default`."

Unless you feel strongly that what you have is broadly consistent in 4.6, please match the 4.9 style.

Reply:
Across 4.6, `Default: x value` is the standard used, hence I chose to stick to it.